src/css/App.css

```diff
@@ -76,8 +78,17 @@ body {
 }
```

Comment:
Great job @itsmraga-hub 🤝

- [ ] [Optional] Kindly remove all commented code from your `ButtonPrimay.js` file and your `App.css` file. Leaving commented code in your project is regarded as a bad practice.

Reply:
Thanks for the approval and for spotting that. On it now.